deploy-manage/deploy/cloud-enterprise/ece-overview.md

```diff
@@ -0,0 +1,44 @@
+---
+navigation_title: ECE Overview
+applies:
+  ece: all
+mapped_urls:
+  - https://www.elastic.co/guide/en/cloud-enterprise/current/Elastic-Cloud-Enterprise-overview.html
+---
+
+# Elastic Cloud Enterprise overview [Elastic-Cloud-Enterprise-overview]
```

Comment:
IMO I'd just pull this up a level because there's not that much stuff

I'd also just merge the service-oriented arch page and the services as docker containers page because they are overlapping

Reply:
this can wait for later though for time reasons

Reply:
done!